modules/thunder_media/thunder_media.install

```diff
@@ -331,6 +306,43 @@ function thunder_media_update_8010() {
     $updateLogger->warning($message);
   }
 
+  // Expected settings for blazy module.
+  $expectedBlazySettings['responsive_image'] = FALSE;
+
+  // New settings for blazy module.
+  $newBlazySettings['responsive_image'] = TRUE;
+
+  if (\Drupal::moduleHandler()->moduleExists('blazy')
+    && \Drupal::moduleHandler()->moduleExists('responsive_image')
+    && $mediaUpdater->updateConfig('blazy.settings', $newBlazySettings, $expectedBlazySettings)
+  ) {
+    $updateLogger->info('The support for responsive images was activated in the Blazy settings.');
+  }
+
+  // Expected configurations for required Video and Image media fields.
+  $expectedConfigs['core.entity_view_display.media.image.default']['content']['field_image']['settings']['image_style'] = 'media_image';
+  $expectedConfigs['core.entity_view_display.media.image.default']['content']['field_image']['settings']['responsive_image_style'] = '';
+  $expectedConfigs['core.entity_view_display.media.gallery.default']['content']['field_media_images']['settings']['optionset'] = 'default';
+  $expectedConfigs['core.entity_view_display.media.image.slick']['content']['field_image']['settings']['image_style'] = 'gallery';
```

Comment:
When updateing from rc2, this precondition check fails, because the core.entity_view_display.media.image.slick configuration is not imported.
The missing configuration should be imported in a new updatehook. Since the shipped configuration file contains the updated value, this import can be done after this update hook.